hack/local-up-cluster.sh

```diff
@@ -662,24 +661,24 @@ function start_cloud_controller_manager {
       exit 1
     fi
 
-    node_cidr_args=""
+    node_cidr_args=()
     if [[ "${NET_PLUGIN}" == "kubenet" ]]; then
-      node_cidr_args="--allocate-node-cidrs=true --cluster-cidr=10.1.0.0/16 "
+      node_cidr_args=("--allocate-node-cidrs=true" "--cluster-cidr=10.1.0.0/16")
```

Comment:
To make the following fixes work correctly:

In ./hack/local-up-cluster.sh line 674:
      ${node_cidr_args} \
      ^---------------^ SC2086: Double quote to prevent globbing and word splitting.